src/main/java/org/opengis/cite/gmljpx20/core/CoreTests.java

```diff
@@ -451,7 +451,7 @@ public void containsUomByReference() {
             XMLBox auxXmlBox = (XMLBox) xmlBox;
             Document doc = docBuilder.parse(new InputSource(new StringReader(auxXmlBox.xmldata.trim())));
             // Note: Just check doc element for allowed coverage types?
-            Boolean hasGmlCovUOMElems = (Boolean) XMLUtils.evaluateXPath(doc, "//swe:uom:*",
+            Boolean hasGmlCovUOMElems = (Boolean) XMLUtils.evaluateXPath(doc, "//*[local-name()=’swe:uom’]",
```

Comment:
’swe:uom’ is not a valid local-name. `//swe:uom` should be fine. Namespace Binding for 'swe' must be added.